src/vs/editor/contrib/find/findWidget.ts

```diff
@@ -807,6 +809,10 @@ export class FindWidget extends Widget implements IOverlayWidget, IHorizontalSas
 
 		this._matchesCount = document.createElement('div');
 		this._matchesCount.className = 'matchesCount';
+
+		this._matchesCount.setAttribute('role', 'alert');
```

Comment:
`role`: `alert` and `aria-live: assertive` should be equivalelnt based on the [W3 spec](https://www.w3.org/TR/WCAG20-TECHS/ARIA19.html)
Does it still work if we use only one of these?

Reply:
correct, works with either